src/sas/sascalc/data_util/nxsunit.py

```diff
@@ -161,27 +158,28 @@ def standardize_units(unit):
     :return: Unit with known, reduced values
     """
     # Catch ang, angstrom, ANG, ANGSTROM, and any capitalization in between
-    # Replace with 'A'
-    unit = re.sub(r'([Aa]{1})([Nn][Gg]([Ss][Tt][Rr][Oo][Mm]){0,1}(\b)){1}',
-                  'A', unit)
+    # Replace with 'Å'
+    unit = re.sub(r'[Åa]ng(strom)?(s)?', 'Å', unit, flags=re.IGNORECASE)
+    unit = re.sub(r'[A]', 'Å', unit)
```

Comment:
This will substitute all capital A anywhere in the string for Å --- probably not what you want.

The square brackets are not necessary if it is a single letter choice (or re for that matter since a simple string replace would do).

Maybe: `if unit == 'A': unit = 'Å'`?

Reply:
There are special characters for word boundaries, so maybe `'A\b'` is a better expression?  That way `invA` => `invÅ`.